### ISSUE TITLE
GrsciColl - collection codes are case sensitive in search

### ISSUE THREAD

**Issue by MortenHofft** [gbif/registry#334]
zero results for http://api.gbif.org/v1/grscicoll/collection?code=naic
one result for http://api.gbif.org/v1/grscicoll/collection?code=NAIC

Won't that lead to more frustration than joy?

**Comment by MortenHofft** [comment]
@ManonGros what is your feeling on this? I find it frustrating every time I do a search myself.

**Comment by ManonGros** [comment]
I agree, it would be good to make it case insensitive. This type of result is indeed counter intuitive and frustrating.

**Comment by ahakanzn** [comment]
Now lookup is also case insensitive and deployed to prod.
